From e733241fb580f032efbbe684ec35d4c9d27f1595 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Thu, 8 Sep 2016 13:38:39 +0300
Subject: [PATCH] Do not error out if perl is not found or its version is too
 old.

This allows use to avoid depending on perl-native if we're not going
to use it (when api-docs are disabled).

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 2a61d6e..29415bd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -36,7 +36,7 @@ dnl Check for Perl.
 dnl
 AC_PATH_PROG([PERL], [perl])
 if test -z "$PERL"; then
-	AC_MSG_ERROR([perl not found])
+	AC_MSG_WARN([perl not found])
 fi
 
 AC_MSG_CHECKING([if Perl version >= 5.18.0])
@@ -44,7 +44,7 @@ if "$PERL" -e "require v5.18.0"; then
 	AC_MSG_RESULT([yes])
 else
 	AC_MSG_RESULT([no])
-	AC_MSG_ERROR([perl >= 5.18.0 is required for gtk-doc])
+	AC_MSG_WARN([perl >= 5.18.0 is required for gtk-doc])
 fi
 
 dnl
-- 
2.9.3

